UltiSnips/all.snippets

```diff
@@ -107,4 +107,23 @@ snippet uuid "Random UUID" w
 `!p if not snip.c: import uuid; snip.rv = uuid.uuid4()`
 endsnippet
 
+
+#################
+#  Parentheses  #
+#################
+snippet () "Parentheses surround" A
+$1(${2:${VISUAL}})$0
+endsnippet
+
+snippet [] "Bracket surround" A
+$1[${2:${VISUAL}}]$0
+endsnippet
+
+snippet {} "Brace surround" A
+$1{${2:${VISUAL}}}$0
+endsnippet
+
+snippet <> "Angle surround" A
+$1<${2:${VISUAL}}>$0
+endsnippet
```

Comment:
Do these need to be in the ultisnips format? It looks like they could be written in the snipmate format

Reply:
I suppose they could. I'm not as familiar with the snipmate format. Is it preferred for them to be in the snipmate format?

Reply:
Yes, the snipmate format is more widely supported. :)

Reply:
Does snipmate support automatic trigger like ultisnips? That's an important feature here.

If this was to be done for snipmate, what file would it go under? There isn't an `snippets/all.snippets` file.

Reply:
Ah yes! Thanks :)